main.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // @title Benefit Service
 // @version 1.0
-// @description This service returns information about benefits
+// @description This service returns information about lifejourneys
```

Comment:
This should say benefits as its for the benefit service, I assume it was a find all replace mistake